.github/workflows/inspektor-gadget.yml

```diff
@@ -364,13 +364,39 @@ jobs:
     # level: 1
     needs: [test-unit, build-kubectl-gadget, build-local-gadget, build-docker-image]
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s-distro: [minikube-github, aro]
```

Comment:
Could `aro` be disabled if `${{ secrets.OPENSHIFT_SERVER }}` is not set? I tried it on a fork of IG, and I get the following:
![image](https://user-images.githubusercontent.com/27575/159684654-b31ab404-7693-400a-8b6e-036947d56d16.png)

Currently, the contributing guide does not list this environment variable in https://github.com/kinvolk/inspektor-gadget/blob/main/docs/CONTRIBUTING.md#github-actions

Reply:
Absolutely! I will work on this.

Reply:
Done. Notice that secrets cannot be used as `if` condition so I used job output as a workaround. WDYT?